virttest/utils_test/libvirt.py

```diff
@@ -2092,6 +2092,51 @@ def create_nwfilter_xml(params):
         raise exceptions.TestFail("Fail to create nwfilter XML: %s" % detail)
 
 
+def get_channel_info(session, interface):
+    """
+    Get channel parameters of interface in vm.
+    """
```

Comment:
Complete docstring for params and return

Reply:
updated, Thank you!